dashboard/lib/script_constants.rb

```diff
@@ -261,6 +261,9 @@ module ScriptConstants
     CSD5 = "csd5".freeze,
     CSD6 = "csd6".freeze,
 
+    # Creating Apps with Devices
+    DEVICES = "devices".freeze,
```

Comment:
this also looks unnecessary. if a test broke when you did this, can you let me know which one?

Reply:
to be fair I think the comment is misleading -- this list gets merged into the list of all course offerings, which many things in this list are already in. I can owe some cleanup on this file, but in the mean time it would be best to avoid adding to this list if possible.

Reply:
Yes good call thank you! Everything is working great and all tests are passing without these extra bits!

Reply:
cleanup PR: https://github.com/code-dot-org/code-dot-org/pull/49803